templates/jobs.html

```diff
@@ -0,0 +1,82 @@
+{% extends "index.html" %}
+{% block content %}
+
+<h1 class="title">Jobs</h1>
+{% include 'level-info-bar.html' %}
+
+
+<div class="columns">
+    <div class="column is-three-fifths is-offset-one-fifth">
```

Comment:
This is ugly now. I'm not sure how to solve this issue, in full size it's too big. I tried to fix it with max-height, but it compressed it. I'm not sure how to solve it, but right now it's ugly.

Reply:
I tried the same things but couldn't find a better solution. I'll leave it this way for the time being.